libqtile/widget/check_updates.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, **config):
                          "Arch_Sup": ("pacman -Sup", 1),
                          "Arch_yay": ("yay -Qu", 0),
                          "Debian": ("apt-show-versions -u -b", 0),
+                         "Gentoo_eix" : ("EIX_LIMIT=0 eix -u# --world", 0),
```

Comment:
Let's make CI happy again

```suggestion
                         "Gentoo_eix": ("EIX_LIMIT=0 eix -u# --world", 0),
```